### ISSUE TITLE
Description required: enable fallback alt tags

### ISSUE THREAD

**Issue by feastdesignco** [campuspress/accessible-content#1]
Thanks for putting this plugin together.

The description for "Enabling this will add fallback alt tags for all the images that have them." doesn't tell you what this is doing.

Is it attempting to fetch the alt tag associated with the media attachment, when the post image has no alt? Does this work for the classic editor and block editor?

I don't remember seeing this on the plugin launch announcement: https://wptavern.com/campuspress-releases-accessible-content-plugin-in-time-for-global-accessibility-awareness-day

**Comment by vladislavbailovic** [comment]
Hello! This is about the changes added to images in Media > Alt text page. The plugin will also try to process the legacy-inserted images (i.e. images inserted into content before the plugin was active, and/or prior to making a change to an image in Media > Alt text page) and retroactively apply such changes to their already existing markup. While there's a lot of effort done in order to make this work seamlessly, there's still a chance this might not go well on every site. This option allows users to opt-in or out to this behavior.

So, in short, yes, it will attempt to fetch the associated alt for the media attachment in scenarios where that media attachment has already been inserted as an `img` tag without an alt attribute. It works with HTML directly, so it will work with images inserted via block and classic editor.

**Comment by feastdesignco** [comment]
Sorry I wasn't clear about the purpose of the issue - it was a request to add more details directly to the plugin so users better understand what they're enabling. We're looking at recommending this to our users and want to make sure it's usable for them before doing so :)

**Comment by vladislavbailovic** [comment]
Ah I see - yeah, makes total sense now :) The option description there could probably use some improvement. Thanks for the clarification, re-opening.
